<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewEditPermissionScreen</class>
 <widget class="QMainWindow" name="NewEditPermissionScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>473</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="locale">
   <locale language="English" country="Austria"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_cancel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="Austria"/>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>420</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="Austria"/>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_permission">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>{New/Edit} Permission</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_permissionName">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>401</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="Austria"/>
    </property>
    <property name="text">
     <string>Permission Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txb_permissionName">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="Austria"/>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_errorPermissionName">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>401</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="Austria"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Permission Name is taken!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="chb_manageUsers">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Manage Users</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chb_storeItems">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Store Items</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chb_withdrawItems">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Withdraw Items</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chb_deleteStorageSlot">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>330</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete Storage Slot</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chb_moveStorageSlot">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>210</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Move Storage Slot</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chb_managePartitions">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>270</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Manage Partitions</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chb_correctQuantity">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>330</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Correct Quantity</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chb_createInventoryReport">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Create Inventory Report</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
